tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -51,3 +53,56 @@ setup_file() {
     ! test -d ~/.fvm
     assert_success
 }
+
+@test "Install Fluvio at 0.10.15" {
+    run $FVM_BIN self install
+    assert_success
+
+    run $FVM_BIN install 0.10.15
+    assert_success
+
+    # Ensure the stable version dir is available
+    test -d ~/.fvm/versions/0.10.15
```

Comment:
since version number would be change in future, make version number configurable instead of hard coding

Reply:
This is still not resolved.  Make version configurable so that CI doesn't break next time version is upgraded